frontend/src/components/SearchForm.js

```diff
@@ -3,7 +3,7 @@ import '../sass/components/searchform.scss';
 import '../sass/base/typography.scss';
 import icon from '../assets/search-icon.png';
 
-const SearchForm = () => (
+const SearchForm = ({radioDifficulty, setRadioDifficulty}) => (
     <Fragment>
          <div class="search-form">
```

Comment:
It was difficulty to notice but if you open your console you should still see it there. In react, to avoid reserved name clashes, css `class` is `className`. So, you cannot use the class keyword here, but `className.` Try to change it, everywhere it appears.

Reply:
You are right, of course we should be using className! I should be checking for console errors more often.